.travis.yml

```diff
@@ -19,12 +19,21 @@ matrix:
       env: TRANSPORT=jackrabbit COMPOSER_FLAGS="--prefer-lowest"
     - php: 7.1
       env: TRANSPORT=doctrine_dbal SYMFONY_VERSION="3.3" PHPBENCH="yes"
+    - php: 7.1
+      env: TRANSPORT=doctrine_dbal SYMFONY_VERSION="3.4" PHPBENCH="yes"
```

Comment:
I think you schould use `^3.4` or `3.4.*`, can you try that?

Reply:
@ElectricMaxxx  Yeah, I'll give it a try!